In [1]:
import os
import pandas as pd
covfilepath = r'G:\Data analysis\datasets\DownloadedDatasets\Covid19\Covid19_27052020_raw_data.csv'
dfraw = pd.read_csv(covfilepath, parse_dates=True)


In [3]:
dfraw.shape

(28183, 21)

In [6]:
dfraw.loc[dfraw.Nationality =='India'].shape

(2658, 21)

In [7]:
dfraw.loc[dfraw.Nationality =='India']

Patient Number State Patient Number Date Announced  \
0                 1.0             KL-TS-P1     30/01/2020   
1                 2.0             KL-AL-P1     02/02/2020   
2                 3.0             KL-KS-P1     03/02/2020   
3                 4.0                DL-P1     02/03/2020   
4                 5.0                TS-P1     02/03/2020   
...               ...                  ...            ...   
24836         24779.0                  NaN     25/04/2020   
24837         24780.0                  NaN     25/04/2020   
24838         24781.0            KL-KL-P12     25/04/2020   
24839         24782.0            KL-KL-P13     25/04/2020   
24840         24783.0            KL-KL-P14     25/04/2020   

       Estimated Onset Date Age Bracket Gender             Detected City  \
0                       NaN          20      F                  Thrissur   
1                       NaN         NaN    NaN                 Alappuzha   
2                       NaN         NaN    NaN                 Kasaragod   
3                       NaN          45      M  East Delhi (Mayur Vihar)   
4                       NaN          24      M                 Hyderabad   
...                     ...         ...    ...                       ...   
24836                   NaN          60      F               Panachikkad   
24837                   NaN           7    NaN              Sasthamkotta   
24838                   NaN          51      M              Kulathupuzha   
24839                   NaN          47      F               Chathannoor   
24840                   NaN          20      M               Peringathur   

      Detected District Detected State State code  ...  \
0              Thrissur         Kerala         KL  ...   
1             Alappuzha         Kerala         KL  ...   
2             Kasaragod         Kerala         KL  ...   
3            East Delhi          Delhi         DL  ...   
4             Hyderabad      Telangana         TG  ...   
...                 ...            ...        ...  ...   
24836          Kottayam         Kerala         KL  ...   
24837            Kollam         Kerala         KL  ...   
24838            Kollam         Kerala         KL  ...   
24839            Kollam         Kerala         KL  ...   
24840            Kannur         Kerala         KL  ...   

                                                   Notes  \
0                                   Travelled from Wuhan   
1                                   Travelled from Wuhan   
2                                   Travelled from Wuhan   
3                          Travelled from Austria, Italy   
4      Travelled from Dubai to Bangalore on 20th Feb,...   
...                                                  ...   
24836                     Mother of health professional    
24837                          Travel history to Sharjah   
24838             Contact transmission, friend of P19074   
24839      Details awaited, ASHA Worker, random sampling   
24840                              Contact transmission    

      Contracted from which Patient (Suspected) Nationality  \
0                                           NaN       India   
1                                           NaN       India   
2                                           NaN       India   
3                                           NaN       India   
4                                           NaN       India   
...                                         ...         ...   
24836                                    P21710       India   
24837                                       NaN       India   
24838                                    P19074       India   
24839                                       NaN       India   
24840                                       NaN       India   

      Type of transmission Status Change Date  \
0                 Imported         14/02/2020   
1                 Imported         14/02/2020   
2                 Imported         14/0

To find out the status of community transmission, the type of transmission should be local. 

In [10]:
dfraw.loc[(dfraw.Nationality =='India')&(dfraw['Type of transmission']=='Local')]

Patient Number State Patient Number Date Announced  \
21               22.0                  NaN     04/03/2020   
22               23.0                  NaN     04/03/2020   
23               24.0                  NaN     04/03/2020   
24               25.0                  NaN     04/03/2020   
25               26.0                  NaN     04/03/2020   
...               ...                  ...            ...   
23320         23263.0                  NaN     24/04/2020   
23321         23264.0                  NaN     24/04/2020   
24836         24779.0                  NaN     25/04/2020   
24838         24781.0            KL-KL-P12     25/04/2020   
24840         24783.0            KL-KL-P14     25/04/2020   

       Estimated Onset Date Age Bracket Gender Detected City  \
21                      NaN         NaN      M          Agra   
22                      NaN         NaN      F          Agra   
23                      NaN          45      F          Agra   
24                      NaN         NaN      M          Agra   
25                      NaN          16      M          Agra   
...                     ...         ...    ...           ...   
23320                   NaN          14      F      Chengala   
23321                   NaN          26      F       Chemnad   
24836                   NaN          60      F   Panachikkad   
24838                   NaN          51      M  Kulathupuzha   
24840                   NaN          20      M   Peringathur   

      Detected District Detected State State code  ...  \
21                 Agra  Uttar Pradesh         UP  ...   
22                 Agra  Uttar Pradesh         UP  ...   
23                 Agra  Uttar Pradesh         UP  ...   
24                 Agra  Uttar Pradesh         UP  ...   
25                 Agra  Uttar Pradesh         UP  ...   
...                 ...            ...        ...  ...   
23320         Kasaragod         Kerala         KL  ...   
23321         Kasaragod         Kerala         KL  ...   
24836          Kottayam         Kerala         KL  ...   
24838            Kollam         Kerala         KL  ...   
24840            Kannur         Kerala         KL  ...   

                                        Notes  \
21                       Family members of P4   
22                       Family members of P4   
23                       Family members of P4   
24                       Family members of P4   
25                       Family members of P4   
...                                       ...   
23320                    Contact transmission   
23321                    Contact transmission   
24836          Mother of health professional    
24838  Contact transmission, friend of P19074   
24840                   Contact transmission    

      Contracted from which Patient (Suspected) Nationality  \
21                                           P4       India   
22                                           P4       India   
23                                           P4       India   
24                                           P4       India   
25                                           P4       India   
...                                         ...         ...   
23320                                       NaN       India   
23321                                       NaN       India   
24836                                    P21710       India   
24838                                    P19074       India   
24840                                       NaN       India   

      Type of transmission Status Change Date  \
21                   Local         04/03/2020   
22                   Local         15/03/2020   
23                   Local         15/03/2020   
24                   Local         15/03/2020   
25                   Local         15/03/2020   
...                    ...                ...   
23320                Local         24/04/2020   
23321                Local         24/04/2020   
24836                

It turns out that out of 2658 patient records we have above in line 6, 705 have had local transmission. Above 27% of infected people are infected locally. 1 in 4 of all infected people have been infected locally.

In [ ]:

dfraw.loc[(dfraw.Nationality =='India')&(dfraw['Type of transmission']=='Local')]

In [ ]:
Cases reported from Mysore.

In [17]:
dfraw.loc[dfraw['Detected District']=='Mysuru']

Patient Number State Patient Number Date Announced  \
324             325.0               KA-P20     21/03/2020   
687             689.0               KA-P52     26/03/2020   
1024           1013.0               KA-P77     29/03/2020   
1025           1014.0               KA-P78     29/03/2020   
1026           1015.0               KA-P79     29/03/2020   
...               ...                  ...            ...   
19030         18973.0              KA-P416     21/04/2020   
19031         18974.0              KA-P417     21/04/2020   
20553         20496.0              KA-P426     22/04/2020   
20554         20497.0              KA-P427     22/04/2020   
24906         24849.0              KA-P497     25/04/2020   

       Estimated Onset Date Age Bracket Gender Detected City  \
324                     NaN          35      M        Mysuru   
687                     NaN          35      M           NaN   
1024                    NaN          39      M     Nanjangud   
1025                    NaN          38      M     Nanjangud   
1026                    NaN          21      M     Nanjangud   
...                     ...         ...    ...           ...   
19030                   NaN          26      M   Nanjanagudu   
19031                   NaN          26      M   Nanjanagudu   
20553                   NaN          56      M   Nanjanagudu   
20554                   NaN          32      M   Nanjanagudu   
24906                   NaN          50      F    Nanjangudu   

      Detected District Detected State State code  ...  \
324              Mysuru      Karnataka         KA  ...   
687              Mysuru      Karnataka         KA  ...   
1024             Mysuru      Karnataka         KA  ...   
1025             Mysuru      Karnataka         KA  ...   
1026             Mysuru      Karnataka         KA  ...   
...                 ...            ...        ...  ...   
19030            Mysuru      Karnataka         KA  ...   
19031            Mysuru      Karnataka         KA  ...   
20553            Mysuru      Karnataka         KA  ...   
20554            Mysuru      Karnataka         KA  ...   
24906            Mysuru      Karnataka         KA  ...   

                                                   Notes  \
324                   Travelled from Dubai via Bengaluru   
687    Had been in contact with healthcare profession...   
1024   All the patients were contacts of Patient-52 (...   
1025   All the patients were contacts of Patient-52 (...   
1026   All the patients were contacts of Patient-52 (...   
...                                                  ...   
19030                                  Contact of KA-P52   
19031                                  Contact of KA-P52   
20553                                 Contact of KA-P383   
20554                                  Contact of KA-P52   
24906                                 Contact of KA-P382   

      Contracted from which Patient (Suspected) Nationality  \
324                                         NaN       India   
687                                         NaN       India   
1024                                       P689       India   
1025                                       P689       India   
1026                                       P689       India   
...                                         ...         ...   
19030                                      P689         NaN   
19031                                      P689         NaN   
20553                                    P14724         NaN   
20554                                      P689         NaN   
24906                                    P14723         NaN   

      Type of transmission Status Change Date  \
324               Imported         07/04/2020   
687                    TBD         26/03/2020   
1024                 Local         28/03/2020   
1025                 Local         12/04/2020   
1026                 Local         12/04/2020   
...                    ...          